service/db.go

```diff
@@ -190,14 +190,32 @@ func insertToken(uuid string, token string) error {
 	return nil
 }
 
+func testUserExist(uuid string) (bool, error) {
+	if uuid == "" {
+		return false, errInvalidUUID
+	}
+
+	command := `SELECT uuid FROM user_svc.accounts WHERE EXISTS (SELECT 1 from user_svc.accounts where uuid = $1)`
+	result, err := postgresDB.Query(command, uuid)
+
```

Comment:
I think we can combine lines 198-199 to the following:
```
result, err := postgresDB.Query("SELECT uuid FROM user_svc.accounts  WHERE uuid = $1", uuid)
```
The benefits are the following:
- This is closer to the actual SQL query itself
- We don't need a nested query since uuid is a primary key and it is used as an index
- `command` is short-lived and not needed anymore

Reply:
I used var command for readability. Do you still want me to change it?